src/seagulls/cli/_di_container.py

```diff
@@ -28,7 +29,7 @@ class SeagullsDiContainer(DeclarativeContainer):
         LoggingClient,
         verbosity=_logging_verbosity,
     )
-
+    _game_state = Singleton(GameState)
```

Comment:
Looks like you create an instance of `GameState` without setting any of the properties. The boolean is `False` by default but you have `active_scene` set to `GameState` but this instance is not going to have an `active_scene` property until someone clicks on a button. How can we fix this?

Reply:
Attempted to address this by setting the active_scene to `None` by default since there is no active scene running yet when first created. `WindowScene` will take the active_scene it is provided and update the GameState.

Reply:
New approach is a little better but it still leaves the responsibility of updating the two properties to the users of the class. I think we can merge this but would be worth thinking about some more and maybe talking through a bit tonight :)